### ISSUE TITLE
The name `delegate` was already registered by another type. (HotChocolate.Stitching.DelegateDirectiveType)

### ISSUE THREAD

**Issue by rkor1984** [ChilliCream/hotchocolate#3512]
I am trying to stitch one stitched schema from three apis to another api using extension files to extend types.
but getting this error , couldnot  find any related resources to resolve this issue. if Some one can help

`HotChocolate.SchemaException: For more details look at the `Errors` property.

1. The name `delegate` was already registered by another type. (HotChocolate.Stitching.DelegateDirectiveType)

   at HotChocolate.Configuration.TypeRegistry.Register(NameString typeName, RegisteredType registeredType)
   at HotChocolate.Configuration.TypeInitializer.<>c__DisplayClass22_0.<CompleteNames>g__CompleteName|0(RegisteredType registeredType)
   at HotChocolate.Configuration.TypeInitializer.ProcessTypes(TypeDependencyKind kind, Func`2 action)
   at HotChocolate.Configuration.TypeInitializer.CompleteNames(Func`1 schemaResolver)
   at HotChocolate.Configuration.TypeInitializer.Initialize(Func`1 schemaResolver, IReadOnlySchemaOptions options)
   at HotChocolate.SchemaBuilder.Setup.InitializeTypes(SchemaBuilder builder, DescriptorContext context, IBindingLookup bindingLookup, IReadOnlyList`1 types, LazySchema lazySchema)
   at HotChocolate.SchemaBuilder.Setup.Create(SchemaBuilder builder)
   at HotChocolate.SchemaBuilder.Create()
   at HotChocolate.SchemaBuilder.HotChocolate.ISchemaBuilder.Create()
   at HotChocolate.Execution.RequestExecutorResolver.CreateSchemaAsync(NameString schemaName, RequestExecutorSetup options, IServiceProvider serviceProvider, CancellationToken cancellationToken)
   at `HotChocolate.Execution.RequestExecutorResolver.CreateSchemaServicesAsync(NameString` schemaName, RequestExecutorSetup options, CancellationToken cancellationToken)
   at HotChocolate.Execution.RequestExecutorResolver.GetRequestExecutorNoLockAsync(NameString schemaName, CancellationToken cancellationToken)
   at HotChocolate.Execution.RequestExecutorResolver.GetRequestExecutorAsync(NameString schemaName, CancellationToken cancellationToken)
   at HotChocolate.Execution.RequestExecutorProxy.GetRequestExecutorAsync(CancellationToken cancellationToken)
   at HotChocolate.Execution.RequestExecutorProxy.GetSchemaAsync(CancellationToken cancellationToken)
   at HotChocolate.AspNetCore.ToolOptionsFileMiddleware.Invoke(HttpContext context)
   at HotChocolate.AspNetCore.HttpGetSchemaMiddleware.InvokeAsync(HttpContext context)
   at HotChocolate.AspNetCore.HttpMultipartMiddleware.InvokeAsync(HttpContext context)
   at HotChocolate.AspNetCore.HttpPostMiddleware.InvokeAsync(HttpContext context)
   at HotChocolate.AspNetCore.WebSocketSubscriptionMiddleware.InvokeAsync(HttpContext context)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)`

**Comment by bneumann** [comment]
I already get it when I try out the example project from the README.

**Comment by michaelstaib** [comment]
can you point to the readme?

**Comment by rkor1984** [comment]
### This is my code for stiching remoteschema to local query


  var builder = services
                    .AddGraphQLServer()
                    .ModifyRequestOptions(opt => opt.IncludeExceptionDetails = true)
                    .AddQueryType<Query>()
                     .AddProjections()
                     .AddFiltering()
                     .AddAuthorization()
                     .AddSorting();

            StitchingApis(services, configuration, builder);


  private static void StitchingApis(IServiceCollection services, IConfiguration configuration, HotChocolate.Execution.Configuration.IRequestExecutorBuilder builder)
        {
            var graphProviders = configuration.GetSection("GraphProviders").Get<GraphProvider[]>();
            var enabledGraphProviders = graphProviders.Where(p => p.Enabled).ToArray();
           
            foreach (var provider in enabledGraphProviders)
            {
                ConfigureHttpClient(services, provider);
                builder.AddRemoteSchema(provider.SchemaName,provider.IgnoreRootTypes);
            }

            foreach (var provider in enabledGraphProviders)
            {
                if (!string.IsNullOrEmpty(provider.ExtensionsFile))
                    builder.AddTypeExtensionsFromFile($"./GraphQL/{provider.ExtensionsFile}");
            }
            //builder;
        }

        private static void ConfigureHttpClient(IServiceCollection services, GraphProvider graph)
        {
            services.AddHttpClient(graph.SchemaName, (sp, client) =>
            {
                client.BaseAddress = new Uri(graph.Url);

                var context = sp.GetRequiredService<IHttpContextAccessor>().HttpContext;
                if (context == null || !context.Request.Headers.ContainsKey("Authorization"))
                    return;

                var authorization = context.Request.Headers["Authorization"].ToString();
                client.DefaultRequestHeaders.Authorization = AuthenticationHeaderValue.Parse(authorization);
            });
        }

**Comment by bneumann** [comment]
Sure thing:
I tried using this: https://github.com/ChilliCream/hotchocolate/tree/main/src/HotChocolate#getting-started

And then I went to localhost:5000/graphql and basically just adding the url to the playground causes the problem. If I comment the GetCharacter and GetCharacters method out it workds.

I figured it has to do with the new Template version 11. So I switched to Version 10 and there it works. 
So in my case it was the new template which seems to have the problem
